test/test_distributions.py

```diff
@@ -119,6 +119,12 @@
             'scale': Variable(torch.randn(1).abs(), requires_grad=True),
         },
     ]),
+    Example(HalfCauchy, [
```

Comment:
Could you add some examples with nonzero loc, e.g. `loc=-1`?

Reply:
will do